emperor/support_files/vendor/js/three.js-plugins/CanvasRenderer.js

```diff
@@ -112,11 +113,18 @@ THREE.CanvasRenderer = function ( parameters ) {
 	_normal = new THREE.Vector3(),
 	_normalViewMatrix = new THREE.Matrix3();
 
+	/* TODO
```

Comment:
I don't think they are, but if left here merging future patches to this file (from upstream) should be easier.